incubator/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSubscribeFactory.java

```diff
@@ -1066,25 +1707,28 @@ private MessageConsumer newKafkaStream(
         int maximum,
         long traceId,
         long authorization,
-        long affinity)
+        long affinity,
+        String clientId,
+        String topic,
```

Comment:
Lets pass these as `StringFW` instead - likely this change is implicit when other feedback above has been applied..